lib/rubocop/remote_config.rb

```diff
@@ -40,11 +43,13 @@ def request(uri = @uri, limit = 10, &block)
       end
 
       handle_response(http.request(request), limit, &block)
+    rescue SocketError
+      handle_response(NetworkInaccessible.new, limit, &block)
     end
 
     def handle_response(response, limit, &block)
       case response
-      when Net::HTTPSuccess, Net::HTTPNotModified
+      when Net::HTTPSuccess, Net::HTTPNotModified, NetworkInaccessible
```

Comment:
I wasn't totally happy with this list containing both `HTTPResponse`s and `*Error`s, but it was the simplest path. Any other ideas?

After the `rescue` on L46, we do have the option of just doing nothing, but I like `handle_response` being applied consistently.